x-pack/plugins/observability_ai_assistant/public/components/chat/chat_flyout.tsx

```diff
@@ -116,6 +128,16 @@ export function ChatFlyout({
   const handleClickChat = (id: string) => {
     setConversationId(id);
     reloadConversation();
+    onSelectConversation?.(id);
+  };
+
+  const handleConversationUpdate = ({
```

Comment:
Yeah, I think the point of readability is a valid one, certainly. 

I like to define them as functions to prevent recreation of the function on every render for some performance benefits. With the observable / streaming we do these components re-render often.

Reply:
You're not wrapping them in `useCallback`, so they get recreated every render anyway. Creating functions is cheap anyway. I think it's useful only when a component that takes the callback as a prop is memoized.